neon_tts_plugin_coqui_ai/__init__.py

```diff
@@ -56,30 +72,70 @@ def get_tts(self, sentence: str, output_file: str, speaker: Optional[dict] = Non
         # request_gender = speaker.get("gender", "female")
         # request_voice = speaker.get("voice")
 
-        # TODO: Below is an example of a common ambiguous language code; test and implement or remove
-        # # Catch Chinese alt code
-        # if request_lang.lower() == "zh-zh":
-        #     request_lang = "cmn-cn"
+        request_lang = speaker.get("language",  self.lang)
+        synthesizer = self._init_model(request_lang)
 
         to_speak = format_speak_tags(sentence)
         LOG.debug(to_speak)
         if to_speak:
             with stopwatch:
-                pass
-                # TODO: Get TTS audio here
-
+                wav_data = synthesizer.tts(sentence)
             LOG.debug(f"TTS Synthesis time={stopwatch.time}")
 
+            with stopwatch:
+                synthesizer.save_wav(wav_data, output_file)
+            LOG.debug(f"File access time={stopwatch.time}")
+
         return output_file, None
 
+    def _init_model(self, lang):
+        lang_params = self.langs[lang]
+        model_name = lang_params["model"]
+        vocoder_name = lang_params["vocoder"]
+
+        model_path, config_path = self._download_model(model_name)
+        vocoder_path, vocoder_config_path = self._download_model(vocoder_name)
+
+        # create synthesizer
+        synt = Synthesizer(tts_checkpoint=model_path,
```

Comment:
It's a bit of a question 
As I checked some coqui initialized models take 1.5GiB
en-us: 800MiB
pl-pl: 900MiB
So I will add it, but with `__init__(cache=False)` as it can cause memory overflow on some systems

Reply:
Done with https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/pull/1/commits/f06d933b0e20af3b0df3d52fabe02e502df3bb02

Reply:
Sounds like a good solution; in the future we might consider managing loaded languages differently based on config or active users, but this should help response time right now